packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -1,82 +1,55 @@
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/Chart.yaml packages/rancher-monitoring/charts/Chart.yaml
 --- packages/rancher-monitoring/charts-original/Chart.yaml
 +++ packages/rancher-monitoring/charts/Chart.yaml
-@@ -1,22 +1,28 @@
+@@ -1,5 +1,5 @@
  apiVersion: v1
- appVersion: 0.38.1
--description: Provides easy monitoring definitions for Kubernetes services, and deployment
--  and management of Prometheus instances.
-+annotations:
+-description: kube-prometheus-stack collects Kubernetes manifests, Grafana dashboards, and Prometheus rules combined with documentation and scripts to provide easy to operate end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus Operator.
++description: Modifies the upstream Prometheus Operator chart, which collects Kubernetes manifests, Grafana dashboards, and Prometheus rules combined with documentation and scripts to provide easy to operate end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus Operator, and enables Prometheus Adapter on a default Prometheus instance.
```

Comment:
Let's nix the "modifies" part, and just talk about what chart does.

Reply:
Based on this and Rajashree's comment [here](https://github.com/rancher/charts/pull/641#discussion_r487563132) about this description, I'm changing this to:

`Collects several related Helm charts, Grafana dashboards, and Prometheus rules combined with documentation and scripts to provide easy to operate end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus Operator.`